domain/ide/workspace/folder.go

```diff
@@ -182,14 +218,14 @@ func (f *Folder) scan(ctx context.Context, path string) {
 		log.Warn().Str("path", path).Str("method", method).Msg("skipping scan of untrusted path")
 		return
 	}
-	issuesSlice := f.IssuesForFile(path)
-	if issuesSlice != nil {
-		log.Info().Str("method", method).
-			Int("issueSliceLength", len(issuesSlice)).
-			Msgf("Cached results found: Skipping scan for %s", path)
-		f.processResults(snyk.ScanData{Issues: issuesSlice})
-		return
-	}
+	//issuesSlice := f.IssuesForFile(path)
+	//if issuesSlice != nil {
+	//	log.Info().Str("method", method).
+	//		Int("issueSliceLength", len(issuesSlice)).
+	//		Msgf("Cached results found: Skipping scan for %s", path)
+	//	f.processResults(snyk.ScanData{Issues: issuesSlice})
+	//	return
+	//}
```

Comment:
I don't see the logic or necessity to skip a scan, when we already have data. WDYT?